Radzen.Blazor/RadzenTabsItem.razor

```diff
@@ -16,6 +16,14 @@
             else
             {
                 <span class="rz-tabview-title">@Text</span>
+                @if (Tabs.TabCloseClick.HasDelegate)
+                {
+                    <span class="rz-ml-05">
+                        <RadzenButton Click="@OnTabCloseClick" class="rz-border-radius" Icon="close" Variant="Variant.Text"
+                                      ButtonStyle="ButtonStyle.Dark" Shade="Shade.Darker" Size="ButtonSize.ExtraSmall"
```

Comment:
Having both ButtonStyle.Dark and Shade.Darker may look odd in a dark theme. What if those are not set at all?